addons/account_analytic_default/account_analytic_default.py

```diff
@@ -1,49 +1,31 @@
-#    Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
-import time
-
-from openerp.osv import fields, osv
-
-class account_analytic_default(osv.osv):
+
+from openerp import api, fields, models
+
+
+class AccountAnalyticDefault(models.Model):
+
     _name = "account.analytic.default"
     _description = "Analytic Distribution"
     _rec_name = "analytic_id"
     _order = "sequence"
-    _columns = {
-        'sequence': fields.integer('Sequence', help="Gives the sequence order when displaying a list of analytic distribution"),
-        'analytic_id': fields.many2one('account.analytic.account', 'Analytic Account'),
-        'product_id': fields.many2one('product.product', 'Product', ondelete='cascade', help="Select a product which will use analytic account specified in analytic default (e.g. create new customer invoice or Sales order if we select this product, it will automatically take this as an analytic account)"),
-        'partner_id': fields.many2one('res.partner', 'Partner', ondelete='cascade', help="Select a partner which will use analytic account specified in analytic default (e.g. create new customer invoice or Sales order if we select this partner, it will automatically take this as an analytic account)"),
-        'user_id': fields.many2one('res.users', 'User', ondelete='cascade', help="Select a user which will use analytic account specified in analytic default."),
-        'company_id': fields.many2one('res.company', 'Company', ondelete='cascade', help="Select a company which will use analytic account specified in analytic default (e.g. create new customer invoice or Sales order if we select this company, it will automatically take this as an analytic account)"),
-        'date_start': fields.date('Start Date', help="Default start date for this Analytic Account."),
-        'date_stop': fields.date('End Date', help="Default end date for this Analytic Account."),
-    }
-
-    def account_get(self, cr, uid, product_id=None, partner_id=None, user_id=None, date=None, company_id=None, context=None):
+
+
+    sequence = fields.Integer(string='Sequence', default='0', help="Gives the sequence order when displaying a list of analytic distribution")
```

Comment:
You can remove the string parameter here, as the ORM will generate a capitalized version of the field name by default.